image-comp/image.html

```diff
@@ -0,0 +1,144 @@
+                        <ul>
+                            <li>
+                                <a href="#">Alert</a>
+                            </li>
+                            <li>
+                                <a href="#">Avatar</a>
+                            </li>
+                            <li>
+                                <a href="#">Badge</a>
+                            </li>
+                            <li>
+                                <a href="#">Button</a>
+                            </li>
+                            <li>
+                                <a href="#">Card</a>
+                            </li>
+                            <li>
+                                <a href="#">Image</a>
+                            </li>
+                            <li>
+                                <a href="#">Input</a>
+                            </li>
+                            <li>
+                                <a href="#">Nevigation</a>
+                            </li>
+                            <li>
+                                <a href="#">Toast</a>
+                            </li>
+
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="aside-body-content">
+            <div class="aside-body heading">
+                <h2>Responsive Image</h2>
+            </div>
+            <div class="aside-body para">
+                <p>
+                    Responsive Image Components are used to dsiplay Images on websites, which are screen responsive.
+                    They expand upto the
+                    width of the container they are in.
+                </p>
+
+            </div>
+            <div class="aside-body heading">
+                <h2> Circular Image</h2>
+            </div>
+            <div class="aside-body para">
+                <p>
+                    Round Images are the circular, with a default dimension of 150px X 150px.
+                </p>
+            </div>
+            <div class="avatar-header">
+                <img src="/image/ded.jpg" alt="image" class="rd-img" />
+                <img src="/image/iron.png" alt="image" class="rd-img" />
+                <img src="/image/image3.jpg" alt="image" class="rd-img" />
+            </div>
+            <div class="aside-body heading">
+                <h2> Square Image</h2>
+            </div>
+            <div class="aside-body para">
+                <p>
+                Square Images are the square shaped, with a default dimension of 150px X 150px.
+                </p>
+            </div>
+            <div class="avatar-header">
+            <img src="/image/ded.jpg" alt="image" class="sq-img" />
+            <img src="/image/iron.png" alt="image" class="sq-img" />
+            <img src="/image/image3.jpg" alt="image" class="sq-img" />
+            </div>
+            <div class="aside-body heading">
```

Comment:
try to avoid using jpg or png , try using webp ( for web optimization). just a suggestion!!!